tools/go.mod

```diff
@@ -34,7 +34,7 @@ require (
 	sigs.k8s.io/yaml v1.4.0
 )
 
-replace sigs.k8s.io/controller-tools => github.com/openshift/controller-tools v0.12.1-0.20241105174925-ead1ef573704
+replace sigs.k8s.io/controller-tools => github.com/JoelSpeed/controller-tools v0.6.3-0.20250106114547-5b11e5175d3d
```

Comment:
temporary?

Reply:
Yeah, need to merge the tools PR first and then will get this fixed up